pkg/privilege/privileges/cache.go

```diff
@@ -2079,7 +2162,14 @@ func (h *Handle) Get() *MySQLPrivilege {
 func (h *Handle) UpdateAll() error {
 	logutil.BgLogger().Warn("update all called")
```

Comment:
Since this is a WARN log and `DefTiDBAccelerateUserCreationUpdate` is false, TiDB would always print this log even if UpdateAll only loads tens or hundreds of items in MySQLPrivilege.
Maybe we can only print this WARN if the number of items exceeds a threshold like `1024` after `priv.LoadAll()`